tests/integration_tests/validation/conformance_suite_config.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class ConformanceSuiteConfig:
+    additional_downloads: dict[str,str] = None
+    args: list[str] = frozenset()
+    capture_warnings: bool = True
+    expected_empty_testcases: frozenset[str] = frozenset()
+    expected_failure_ids: frozenset[str] = frozenset()
+    expected_model_errors: frozenset[str] = frozenset()
+    extract_path: str = None
+    file: str = None
+    info_url: str = None
+    local_filepath: str = None
+    membership_url: str = None
```

Comment:
Somewhat arbitrary line.
I have a separate branch that allows users to provide XBRL International username/pass as command line options to also download the private conformance suites automatically. I opted to separate that out as there were some complications, the biggest of which being that I don't have credentials to actually test this out. I also recall that, while working on getting some of the private CS zips ready for CI, some manual work was needed to prepare them for actual use here. If that's still the case, it won't be trivial to automate that preparation.